src/styles.scss

```diff
@@ -145,3 +145,7 @@ body {
     color: rgb(0, 0, 0, 0.25) !important;
   }
 }
+
+.cdk-overlay-pane {
+  width: unset !important;
+}
```

Comment:
Are you sure this rule has no undesirable effect on other components ?

Reply:
You are right! The map preview doesn't render correclty anymore. I'll try another solution

Reply:
It can be a good idea to keep the "..." and add a tooltip to display the whole text

Reply:
done